docs/custom_gates.ipynb

```diff
@@ -87,7 +87,7 @@
     "id": "y8P1T6duC-yo"
```

Comment:
<p>So I can add a comment from here...let's see how that looks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/balopat/Cirq/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>